docs/tasks/access-application-cluster/create-external-load-balancer.md

```diff
@@ -8,18 +8,24 @@ redirect_from:
 * TOC
```

Comment:
The template will automatically take care of adding a TOC, so you can remove lines 8 and 9.